packages/nodes-base/nodes/Orbit/Orbit.node.ts

```diff
@@ -0,0 +1,326 @@
+						body.occurred_at = additionalFields.occurredAt as string;
+					}
+
+					responseData = await orbitApiRequest.call(this, 'POST', `/${workspaceId}/members/${memberId}/activities`, body);
+				}
+				if (operation === 'get') {
+					const workspaceId = this.getNodeParameter('workspaceId', i) as string;
+					const memberId = this.getNodeParameter('memberId', i) as string;
+
+					responseData = await orbitApiRequest.call(this, 'GET', `/${workspaceId}/members/${memberId}/activities`);
+				}
+				if (operation === 'getAll') {
+					const workspaceId = this.getNodeParameter('workspaceId', i) as string;
+
+					responseData = await orbitApiRequest.call(this, 'GET', `/${workspaceId}/activities`);
+				}
+			}
+			if (resource === 'member') {
+				if (operation === 'delete') {
+					const workspaceId = this.getNodeParameter('workspaceId', i) as string;
+					const memberId = this.getNodeParameter('memberId', i) as string;
+					responseData = await orbitApiRequest.call(this, 'DELETE', `/${workspaceId}/members/${memberId}`);
+				}
+				if (operation === 'get') {
+					const workspaceId = this.getNodeParameter('workspaceId', i) as string;
+					const memberId = this.getNodeParameter('memberId', i) as string;
+					responseData = await orbitApiRequest.call(this, 'GET', `/${workspaceId}/members/${memberId}`);
+				}
+				if (operation === 'getAll') {
+					const workspaceId = this.getNodeParameter('workspaceId', i) as string;
+					responseData = await orbitApiRequest.call(this, 'GET', `/${workspaceId}/members`);
+				}
+				if (operation === 'lookup') {
+					const workspaceId = this.getNodeParameter('workspaceId', i) as string;
+					const githubUsername = this.getNodeParameter('githubUsername', i) as string;
+					responseData = await orbitApiRequest.call(this, 'GET', `/${workspaceId}/github_user/${githubUsername}`);
+				}
+				if (operation === 'update') {
+					const workspaceId = this.getNodeParameter('workspaceId', i) as string;
+					const memberId = this.getNodeParameter('memberId', i) as string;
+					const additionalFields = this.getNodeParameter('additionalFields', i) as IDataObject;
+					const body: IDataObject = {
+					};
+					if (additionalFields.bio) {
+						body.bio = additionalFields.bio as string;
+					}
+					if (additionalFields.birthday) {
+						// This isn't functioning at the moment
+						body.birthday = additionalFields.birthday as string;
+					}
+					if (additionalFields.company) {
+						body.company = additionalFields.company as string;
+					}
+					if (additionalFields.devTo) {
+						body.devto = additionalFields.devTo as string;
+					}
+					if (additionalFields.linkedin) {
+						body.linkedin = additionalFields.linkedin as string;
+					}
+					if (additionalFields.location) {
+						body.location = additionalFields.location as string;
+					}
+					if (additionalFields.name) {
+						body.name = additionalFields.name as string;
+					}
+					if (additionalFields.orbitLevel) {
```

Comment:
Orbit level can be read but not written to. Do we still keep it in the definition?